README.md

```diff
@@ -48,11 +48,17 @@ var childWalletSecret = lib.derivePublic(parentWalletPublicKey, 1, 1)
 * `Buffer toPublic(Buffer privateKey)`
 * `Buffer decodePaperWalletMnemonic(string paperWalletMnemonic)`
 * `Buffer xpubToHdPassphrase(Buffer xpub)`
-* `string packAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
+* `Buffer deriveBootstrapAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
```

Comment:
the original comment about the "derive" verb use was intended to suggest unifying it to "pack" and we can have packByronAddress as a replacement of the original packAddress and rename those "derive..." operations into "pack" as well as the "pack" verb fits more since no derivation is happening in the function